Antorus/Locales/ruRU.lua

```diff
@@ -1,82 +1,82 @@
 local L = BigWigs:NewBossLocale("Argus the Unmaker", "ruRU")
 if not L then return end
 if L then
-	--L.combinedBurstAndBomb = "Combine Soulburst and Soulbomb"
-	--L.combinedBurstAndBomb_desc = "|cff71d5ffSoulbombs|r are always applied in combination with |cff71d5ffSoulbursts|r. Enable this option to combine those two messages into one."
+	L.combinedBurstAndBomb = "Одновременные Бомба души и Взрывная душа"
+	L.combinedBurstAndBomb_desc = "|cff71d5ffБомбы души|r всегда накладываются одновременно с |cff71d5ffВзрывными душами|r. Включите эту опцию, чтобы объединить два сообщения в одно."
 
-	--L.custom_off_always_show_combined = "Always show the combined Soulburst and Soulbomb message"
-	--L.custom_off_always_show_combined_desc = "The combined message won't be displayed if you get the |cff71d5ffSoulbomb|r or the |cff71d5ffSoulburst|r. Enable this option to always show the combined message, even when you're affected. |cff33ff99Useful for raid leaders.|r"
+	L.custom_off_always_show_combined = "Всегда отображать объединенное сообщение о Взрывной душе и Бомбе души"
+	L.custom_off_always_show_combined_desc = "Объединенное сообщение не будет отображено, если на Вас накладывается |cff71d5ffБомба души|r or the |cff71d5ffВзрывная душа|r. Включите данную опцию, если вы хотите всегда видеть объединённое сообщение. |cff33ff99Полезно для лидера рейда.|r"
 
-	--L.fear_help = "Sargeras' Fear Combination"
-	--L.fear_help_desc = "Say a special message if you're afflicted by |cff71d5ffSargeras' Fear|r and |cff71d5ffSoulblight|r/|cff71d5ffSoulburst|r/|cff71d5ffSoulbomb|r/|cff71d5ffSentence of Sargeras|r."
+	L.fear_help = "Страх перед Саргерасом Комбинация"
+	L.fear_help_desc = "Сказать специальное сообщение, если на вас |cff71d5ffСтрах перед Саргерасом|r и |cff71d5ffИзнуряющая чума|r/|cff71d5ffВзрывная душа|r/|cff71d5ffБомба души|r/|cff71d5ffПриговор Саргераса|r."
 
-	--L[257931] = "Fear" -- short for Sargeras' Fear
-	--L[248396] = "Blight" -- short for Soulblight
-	--L[251570] = "Bomb" -- short for Soulbomb
-	--L[250669] = "Burst" -- short for Soulburst
-	--L[257966] = "Sentence" -- short for Sentence of Sargeras
+	L[257931] = "Страх" -- short for Sargeras' Fear
+	L[248396] = "Чума" -- short for Soulblight
+	L[251570] = "Бомба" -- short for Soulbomb
+	L[250669] = "Взрыв" -- short for Soulburst
+	L[257966] = "Приговор" -- short for Sentence of Sargeras
 
-	--L.stage2_early = "Let the fury of the sea wash away this corruption!"
-	--L.stage3_early = "No hope. Just pain. Only pain!"
+	L.stage2_early = "Ярость морей смоет эту скверну!"  -- Aman'thul's accurate quote
+	L.stage3_early = "Надежды нет. Есть только боль!" -- Argus' accurate quote 
 
-	--L.gifts = "Gifts: %s (Sky), %s (Sea)"
-	--L.burst = "|T1778229:15:15:0:0:64:64:4:60:4:60|tBurst:%s" -- short for Soulburst
-	--L.bomb = "|T1778228:15:15:0:0:64:64:4:60:4:60|tBomb (%d):|T137002:0|t%s - " -- short for Soulbomb
+	L.gifts = "ДарыGifts: %s (Небо), %s (Море)"
```

Comment:
Looks like you forgot to remove "Gifts".

Reply:
Indeed